<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="FormWallet" name="wdt_Wallet" native="true"/>
    </item>
    <item>
     <widget class="FormInvestment" name="wdt_Investment" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1224</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionWallet"/>
    <addaction name="actionMatureInvestment"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionWallet">
   <property name="text">
    <string>Wallet</string>
   </property>
  </action>
  <action name="actionMatureInvestment">
   <property name="text">
    <string>Investment</string>
   </property>
  </action>
  <action name="action_Theme">
   <property name="text">
    <string>&amp;Themes</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormWallet</class>
   <extends>QWidget</extends>
   <header>UI/Formwallet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormInvestment</class>
   <extends>QWidget</extends>
   <header>UI/Forminvestment.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
